### PR TITLE
feat: add notebook for experiments autologging

### PR DESCRIPTION
**Pull request by inardini**
<br>
Adding a notebook about how automate ML experiment tracking with Vertex AI Experiments Autologging. 
<br>

If you are opening a PR for `Official Notebooks` under the [notebooks/official](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/official) folder, follow this mandatory checklist:
- [x] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [x] Follow the style and grammar rules outlined in the above notebook template.
- [x] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [x] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).
- [x] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [x] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/CODEOWNERS) file under the `Official Notebooks` section, pointing to the author or the author's team.
- [x] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

### CODE REVIEW

**Comment by andrewferlitsch** [comment]
Step #5: Step #4: Cell In[15], line 12, in set_seed(seed)
Step #5: Step #4:       9 import random
Step #5: Step #4:      11 import numpy as np
Step #5: Step #4: ---> 12 import tensorflow as tf
Step #5: Step #4:      13 import torch
Step #5: Step #4:      15 random.seed(seed)
Step #5: Step #4: 
Step #5: Step #4: ModuleNotFoundError: No module named 'tensorflow'

File: notebooks/official/experiments/get_started_with_vertex_experiments_autologging.ipynb [hunk 1]
```diff
@@ -0,0 +1,1230 @@
+{
```
Comment (andrewferlitsch) [inline notebooks/official/experiments/get_started_with_vertex_experiments_autologging.ipynb:1]: <p>Data science -&gt; data science</p><p>you may want -&gt; you</p><p>Vertex Experiments -&gt; Vertex AI Experiments</p><p>Experiment Runs -&gt; experiment runs</p><p> </p><p>End the Overview with a tag/linkback.</p><p> </p><p><code>Learn more about \[Autologging data to an experiment run\](</code>https://cloud.google.com/vertex-ai/docs/experiments/autolog-data<code>)</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1574/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (andrewferlitsch) [inline notebooks/official/experiments/get_started_with_vertex_experiments_autologging.ipynb:1]: <p>autologging -&gt; Autologging</p><p> </p><p>services used:</p><ul><li>Vertex AI Experiments</li></ul><p> </p><p>Sklearn -&gt; scikit-learn</p><p>Pytorch -&gt; PyTorch</p><p> </p><p>has been logged -&gt; are logged</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1574/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Comment (andrewferlitsch) [inline notebooks/official/experiments/get_started_with_vertex_experiments_autologging.ipynb:1]: <p>replace --user with</p><p> </p><p>USER=''</p><p>! pip3 install {USER} ...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1574/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Comment (andrewferlitsch) [inline notebooks/official/experiments/get_started_with_vertex_experiments_autologging.ipynb:1]: <p>official Google Cloud bucket -&gt; public Cloud Storage bucket</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1574/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Comment (andrewferlitsch) [inline notebooks/official/experiments/get_started_with_vertex_experiments_autologging.ipynb:1]: <p>library -&gt; SDK</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1574/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>

Comment (andrewferlitsch) [inline notebooks/official/experiments/get_started_with_vertex_experiments_autologging.ipynb:1]: <p>Helpers -&gt; Helper functions</p><p> </p><p>Below you have -&gt; Below you define</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1574/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>

Comment (andrewferlitsch) [inline notebooks/official/experiments/get_started_with_vertex_experiments_autologging.ipynb:1]: <p>combine the 2 code cells</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1574/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>

Comment (andrewferlitsch) [inline notebooks/official/experiments/get_started_with_vertex_experiments_autologging.ipynb:1]: <p>in heading. Model Experimentation -&gt; model experimentation</p><p> </p><p>allows to run -&gt; allows you to run</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1574/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>

Comment (andrewferlitsch) [inline notebooks/official/experiments/get_started_with_vertex_experiments_autologging.ipynb:1]: <p>drop Keras (part of TensorFlow)</p><p> </p><p>Tensorflow -&gt; TensorFlow</p><p>Tensorboard -&gt; TensorBoard</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1574/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='42'/>

Comment (andrewferlitsch) [inline notebooks/official/experiments/get_started_with_vertex_experiments_autologging.ipynb:1]: <p>Tensorflow -&gt; TensorFlow</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1574/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='56'/>

Comment (inardini) [reply notebooks/official/experiments/get_started_with_vertex_experiments_autologging.ipynb:1]: done

Comment (inardini) [reply notebooks/official/experiments/get_started_with_vertex_experiments_autologging.ipynb:1]: done

Comment (inardini) [reply notebooks/official/experiments/get_started_with_vertex_experiments_autologging.ipynb:1]: done

Comment (inardini) [reply notebooks/official/experiments/get_started_with_vertex_experiments_autologging.ipynb:1]: done

Comment (inardini) [reply notebooks/official/experiments/get_started_with_vertex_experiments_autologging.ipynb:1]: done

Comment (inardini) [reply notebooks/official/experiments/get_started_with_vertex_experiments_autologging.ipynb:1]: done

Comment (inardini) [reply notebooks/official/experiments/get_started_with_vertex_experiments_autologging.ipynb:1]: done

Comment (inardini) [reply notebooks/official/experiments/get_started_with_vertex_experiments_autologging.ipynb:1]: done

Comment (inardini) [reply notebooks/official/experiments/get_started_with_vertex_experiments_autologging.ipynb:1]: done

**Comment by andrewferlitsch** [comment]
�[31mERROR: Invalid requirement: 'tensorflow!'�[0m�[31m

**Review by andrewferlitsch — APPROVED** [review]

**Pull request merged**